sdk/src/main/java/com/uid2/EUIDManager.kt

```diff
@@ -0,0 +1,122 @@
+package com.uid2
+
+import android.content.Context
+import com.uid2.UID2Manager.Companion.APPLICATION_ID_DEFAULT
+import com.uid2.network.DefaultNetworkSession
+import com.uid2.network.NetworkSession
+import com.uid2.storage.FileStorageManager
+import com.uid2.storage.FileStorageManager.Store.EUID
+import com.uid2.storage.StorageManager
+import com.uid2.utils.InputUtils
+import com.uid2.utils.Logger
+import com.uid2.utils.TimeUtils
+import kotlinx.coroutines.Dispatchers
+
+public class EUIDManager {
+
+    public companion object {
+
+        public sealed interface Environment {
```

Comment:
Move this interface (and sub classes) out of the companion object (move to within `EUIDManager` instead)